src/vs/editor/contrib/suggest/suggestWidget.ts

```diff
@@ -481,16 +489,10 @@ export class SuggestWidget implements IContentWidget, IVirtualDelegate<ICompleti
 		this.editor.layoutContentWidget(this);
 	}
 
-	private onEditorBlur(): void {
-		if (sticky) {
-			return;
+	private onEditorTextEvent(): void {
+		if (!sticky && !this.details.isFocused()) {
+			this.setState(State.Hidden);
 		}
-
-		this.editorBlurTimeout.cancelAndSet(() => {
```

Comment:
The timeout is actually not being used since [this](https://github.com/Microsoft/vscode/pull/56052/files#diff-1933db8ed5ace374e3217601d164f9b3R112) is always hiding the widget first.

Reply:
That's bug that got introduced recently. See https://github.com/Microsoft/vscode/commit/c0917d8e6f0f1b5ea06ef231b24ff2ffa1416589. I would suggest to remove changes related to the editor blur/focus events in this PR.